src/main/java/frc/robot/subsystems/Drivetrain.java

```diff
@@ -24,15 +30,18 @@ public class Drivetrain extends SubsystemBase {
 
   public Drivetrain() {
     // TODO 1.1.2: Initialize motors
-
+         leftMotor1 = new CANSparkMax(DriveConstants.LEFT_MOTOR_1_ID, MotorType.kBrushless);
+         rightMotor1 = new CANSparkMax(DriveConstants.RIGHT_MOTOR_1_ID, MotorType.kBrushless);
     // TODO 1.1.3: Set motors to brake mode
+      leftMotor1.setIdleMode(IdleMode.kBrake);
+      rightMotor1.setIdleMode(IdleMode.kBrake);
 
     // TODO 1.1.4: Make motor2s follow motor1s
 
     // TODO 1.2.4: Invert motors if necessary
 
     // TODO 2.1.1: Create DifferentialDriveSim if the robot isn't real
-
+      DifferentialDrivetrainSim burrito;
   }
```

Comment:
This should be an instance variable instead of a local variable in the constructor. Also, name it something more descriptive than burrito, like drivetrainSim or anything else that describes what it is.

Reply:
I don't think there's a TODO telling you where to put it, but it should be declared outside the constructor and defined inside the constructor, like the motors. Also, I think the activities document says to define it in an if statement.